embedded/sql/sql_grammar.y

```diff
@@ -769,6 +771,16 @@ opt_not:
         $$ = true
     }
 
+is_op:
```

Comment:
alternatively:
```
is_op:
        IS
        {
            $$ = EQ
        }
|
        IS NOT
        {
             $$ = NE
        }
```

or even

```
    exp IS NULL
    {
        $$ = &CmpBoolExp{left: $1, op: EQ, right: &NullValue{t: AnyType}}
    }
|
    exp IS NOT NULL
    {
        $$ = &CmpBoolExp{left: $1, op: NE, right: &NullValue{t: AnyType}}
    }
```

Reply:
I tried the second option yesterday and couldn't get it to work, must have been some typo, today it works without issues.